stone/lang/parser.py

```diff
@@ -982,6 +983,46 @@ def p_ex_list_items_extend(self, p):
         p[0].append(p[3])
 
     # --------------------------------------------------------------
+    # Maps
+    #
+
+    def p_ex_map(self, p):
+        """ex_map : LBRACE ex_map_pairs RBRACE
+                  | LBRACE empty RBRACE"""
+        p[0] = {} if p[2] is None else p[2]
```

Comment:
Correct, `ex_map_pairs` should never be an empty list.  The second `empty` case is to handle if somebody writes:
```
struct S
    arg Map(String, String)
    
    example
        arg = {}
````